### PR TITLE
Add somatic/germline gene page

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

**Review by jfkonecn — CHANGES_REQUESTED** [review]

File: src/main/webapp/app/components/PageContainer.tsx [hunk 1]
```diff
@@ -1,15 +1,44 @@
...
-      </Col>
-    </Row>
+    <div className={'view-wrapper'}>
+      <div
+        className={
+          inGenePage
+            ? ''
+            : props.windowStore.isXLscreen
+            ? 'container'
+            : 'container-fluid'
```
[the remark this answers is not in the document]
Comment (zhx828) [reply src/main/webapp/app/components/PageContainer.tsx:36]: Good question. Didn't look into, inherited from the original.

File: src/main/webapp/app/Main.tsx [hunk 2]
```diff
@@ -104,15 +104,16 @@ class Main extends React.Component<IMainPage> {
...
-              appStore={this.props.appStore}
-              routing={this.props.routing}
-            />
-          </Container>
-        </div>
+        <PageContainer
+          routing={this.props.routing}
+          windowStore={this.props.windowStore}
+        >
+          <AppRouts
```
[the remark this answers is not in the document]
Comment (zhx828) [reply src/main/webapp/app/Main.tsx:111]: Yup, will fix

File: src/main/webapp/app/components/infoTile/LoETile.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      ),
+    });
+  }
+  if (props.highestDiagnosticImplicationLevel) {
+    categories.push({
+      title: 'Diagnostic',
+      content: (
+        <div>
+          <OncoKBLevelIcon
+            size={'s2'}
```
[the remark this answers is not in the document]
Comment (zhx828) [reply src/main/webapp/app/components/infoTile/LoETile.tsx:49]: you meant why it's not a default?

File: src/main/webapp/app/pages/genePage/SomaticGermlineGenePage.tsx [hunk 4]
```diff
@@ -0,0 +1,948 @@
...
+      >
+        Subscribe Now
+      </Button>
+    </div>
+  );
+};
+
+@inject('appStore', 'windowStore', 'routing')
+@observer
+export default class SomaticGermlineGenePage extends React.Component<
```
[the remark this answers is not in the document]
Comment (zhx828) [reply src/main/webapp/app/pages/genePage/SomaticGermlineGenePage.tsx:128]: We have a store in this class. And we manage some observables states here too.

File: src/main/webapp/app/components/PageContainer.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (zhx828) [reply src/main/webapp/app/components/PageContainer.tsx:36]: For public website, we haven't really used the @media to manage the layout. I would stick to the classname for now.

File: src/main/webapp/app/components/infoTile/LoETile.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (jfkonecn) [reply src/main/webapp/app/components/infoTile/LoETile.tsx:49]: `size={'s2'}` vs `size="s2"`

**Pull request merged**